cap/modules/schemas/models/schemas.py

```diff
@@ -55,19 +58,13 @@
 }
 
 
-class Schema(db.Model):
+class Schema(db.Model, SchemaVersion):
```

Comment:
as discussed, let's not separate this one and keep SchemaVersion fields inside Schema